annotations/fenix/metrics/credit_cards.autofill_prompt_dismissed/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - Autofill
+---
+
+This is a stub commentary for the `credit_card.autofill_prompt_dismissed` metric: please feel free to edit (read the
```

Comment:
This is no longer necessary, I'll remove it before landing but now you know for next time! :)